l1-contracts/test/sparta/Sparta.t.sol

```diff
@@ -64,27 +64,58 @@ contract SpartaTest is DecoderBase {
     }
   }
 
-  function _testProposerForFutureEpoch() public {
-    // @todo Implement
+  function testProposerForNonSetupEpoch(uint8 _epochsToJump) public {
+    uint256 pre = rollup.getCurrentEpoch();
+    vm.warp(block.timestamp + uint256(_epochsToJump) * rollup.EPOCH_SIZE() * rollup.SLOT_SIZE());
```

Comment:
Can we make an issue to rename this to `SLOT_DURATION`?

Reply:
Will update `SLOT_SIZE -> SLOT_DURATION` and `EPOCH_SIZE -> EPOCH_DURATION`